tests/utility/type.ts

```diff
@@ -95,11 +95,12 @@ test('do nothing on disabled element', async () => {
 
 describe('on shadow DOM', () => {
   test('type into an input element', async () => {
-    const {element, user} = setup('<shadow-input></shadow-input>')
+    defineShadowInputCustomElementIfNotDefined()
+    const {element, user} = setup<ShadowInput>('<shadow-input></shadow-input>')
 
-    // Skip click because delegatesFocus is not implemented in jsdom
+    // Skip click because the element is already focused
     await user.type(element, 'test', {skipClick: true})
 
-    expect((element as ShadowInput).value).toEqual('test')
+    expect(element.value).toEqual('test')
   })
 })
```

Comment:
If the test skips the click because it wouldn't work with it, we should add a test describing the missing implementation on `.pointer()` and move the typing test to `.keyboard()`.

Reply:
I removed this, because my primary focus is cut, copy and paste. Maybe this can be part of another PR.

Reply:
But the test was there before. We shouldn't just remove it to merge this.
We need to make sure that we have a good understanding of what is missing and needs to be fixed later if we don't deal with it in this PR.

Reply:
As I said I assumed it was a quick win. It was not, so I removed it for now. It is not part of what this PR describes anyway.